docs/sources/flow/reference/components/remote.http.md

```diff
@@ -133,6 +137,7 @@ set of scrape targets:
 ```river
 remote.http "targets" {
   url = env("MY_TARGETS_URL")
+  method = "GET"
```

Comment:
Let's remove this from the example since it's not a required field, and we're all set!
```suggestion
```